doc/source/whatsnew/v1.2.0.rst

```diff
@@ -856,7 +856,7 @@ Other
 - Bug in :meth:`Index.drop` raising ``InvalidIndexError`` when index has duplicates (:issue:`38051`)
 - Bug in :meth:`RangeIndex.difference` returning :class:`Int64Index` in some cases where it should return :class:`RangeIndex` (:issue:`38028`)
 - Fixed bug in :func:`assert_series_equal` when comparing a datetime-like array with an equivalent non extension dtype array (:issue:`37609`)
-
+- Bug in :func:`.is_bool_dtype` would raise when passed a valid string such as ``"boolean"`` (:issue:`38386`).
```

Comment:
typo extra period before is_bool_dtype

Reply:
The period before `is_bool_dtype` allows sphinx link to the proper page; without it, a link will not be made. However, the period after the issue number is a typo.